tdrs-frontend/src/components/Reports/FRAReports.jsx

```diff
@@ -802,33 +807,39 @@ const FRAReports = () => {
             reportTypeLabel={getReportTypeLabel()}
           />
 
-          {localAlert.active && (
-            <div
-              ref={alertRef}
-              className={classNames('usa-alert usa-alert--slim', {
-                [`usa-alert--${localAlert.type}`]: true,
-              })}
-            >
-              <div className="usa-alert__body" role="alert">
-                <p className="usa-alert__text">{localAlert.message}</p>
-              </div>
-            </div>
+          {!isRegionalStaff && (
```

Comment:
@ADPennington since FRA is now merged, i added this conditional so that regional staff users are similarly prevented from uploading FRA reports. other than this conditional, i added `OFA Regional Staff` to one of the auth selectors - that was the extent of the change. cc @elipe17